Library/Homebrew/cmd/test-bot.rb

```diff
@@ -651,7 +651,8 @@ def cleanup_before
         safe_system "git", "checkout", "-f", "master"
         # This will uninstall all formulae, as long as
         # HOMEBREW_REPOSITORY == HOMEBREW_PREFIX, which is true on the test bots
-        safe_system "git", "clean", "-ffdx", "--exclude=/Library/Taps/" unless ENV["HOMEBREW_RUBY"] == "1.8.7"
+        Tap.names.reject { |t| t == "homebrew/core" }.each { |s| safe_system "brew untap #{s}" }
+        safe_system "git", "clean", "-ffdx" unless ENV["HOMEBREW_RUBY"] == "1.8.7"
```

Comment:
No, you're right, I should have left the exclude in regardless. This is what happens when I throw a PR together 15 minutes before I go to sleep. I'll tweak it in the morning.